src/_imaging.c

```diff
@@ -3688,20 +3630,14 @@ static struct PyMethodDef methods[] = {
     {"chop_hard_light", (PyCFunction)_chop_hard_light, METH_VARARGS},
     {"chop_overlay", (PyCFunction)_chop_overlay, METH_VARARGS},
 
-#endif
-
-#ifdef WITH_UNSHARPMASK
-    /* Kevin Cazabon's unsharpmask extension */
```

Comment:
Just for clarity - you're removing 'Kevin Cazabon' presumably because the code has been modified over time?

Reply:
If the reason were due to modifications over time, I would have included additional names accordingly. The actual reason is that I was unsure of the specific value this information provides in the comments. However, if this is important, I can revert the change.